src/test/java/net/snowflake/client/jdbc/diagnostic/DiagnosticContextLatestIT.java

```diff
@@ -0,0 +1,356 @@
+package net.snowflake.client.jdbc.diagnostic;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import net.snowflake.client.category.TestCategoryDiagnostic;
+import net.snowflake.client.core.SFSessionProperty;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(TestCategoryDiagnostic.class)
```

Comment:
this category is not enrolled in any our CI builds - it means that it will be only for manual tests?

Reply:
I added the TestCategory now to the CI builds.